File: 1071.py
x = int(input())
y = int(input())

soma = int()

if x > y:
    for i in range(y+1, x):

        if i%2 != 0:
            soma += i

if y > x:
    for i in range(x+1, y):

        if i%2 != 0:
            soma += i

print(soma)

File: 1071.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
    
var x = parseInt(lines.shift());
var y = parseInt(lines.shift());
var soma = 0

for (let i = (y + 1); i < x; i++){
    if (i % 2 != 0) soma+= i;
}

console.log(soma);
